src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+spring:
+  application:
+    name: eureka-server-application
+
+server:
+  port: 8083
+eureka:
+  client:
+    registerWithEureka: false
+    fetchRegistry: false
+    service-url:
+      defaultZone: http://localhost:8083/eureka
```

Comment:
What does this do?

Reply:
I should clarify I'm asking about the `defaultZone` property

Reply:
9 prevents Eureka from registering with itself as it would if we were spinning up a half-dozen instances.
10 prevents Eureka from trying to find other Eureka instances to see what they have registered.

The service-url-defaultZone appears to have the same basic function as server-port.

Reply:
We might need to make `defaultZone` an environment variable, but I'm not completely sure about that. I'll play around with it when I work on deploying later

Reply:
I feel really dumb right now, but I uploaded this stuff using drag and drop rather than creating a local repository and then properly connecting it. I am unsure of how to get the target stuff out of this commit given that I don't have a repository set up to commit further changes.

Reply:
I thought that I might be able to manually delete the files if I changed it to a draft, but I was only able to delete a single file (the delete option is grayed out for the other relevant files).

Reply:
My gitignore actually had the target folder in it already, but I completely bypassed that by using the manual process.

Reply:
I'll fix it.